assets/javascripts/kitten/hoc/trigger-enhancer.test.js

```diff
@@ -0,0 +1,264 @@
+import sinon from 'sinon'
+import { expect } from 'chai'
+import { mount, shallow } from 'enzyme'
+import React from 'react'
+import { triggerEnhancer } from 'kitten/hoc/trigger-enhancer'
+import domElementHelper from 'kitten/helpers/dom/element-helper'
+import { localStorageUtils } from 'kitten/helpers/utils/local-storage'
+
+// Because of an unexpected loss of the jsdom context when React re-renders a
+// component (using setState on componentDidMount), we have to mount all
+// enhancedComponents with the `autorun` prop set to false.
+// cf. https://github.com/facebook/react/issues/4025
+
+describe('autoTriggerEnhancer', () => {
+  const sandbox = sinon.sandbox.create()
+  const CustomComponent = (props) => <div>{ props.text }</div>
+  const EnhancedComponent = triggerEnhancer(CustomComponent, {
+    text: 'my custom text'
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe('by default', () => {
+    const enhancedComponent = shallow(<EnhancedComponent />)
+
+    it('renders nothing on mount', () => {
+      expect(enhancedComponent).to.be.blank()
+    })
+  })
+
+  describe('#componentDidMount()', () => {
+    let enhancedComponent
+
+    beforeEach(() => {
+      enhancedComponent = mount(<EnhancedComponent autorun={ false } />)
+    })
+
+    describe('calls shouldStart()', () => {
```

Comment:
Ici ce describe a été nommé comme on appellerait un `it`. Je pense que tu peux te passer du describe et du before ici, et faire juste `it('calls shouldStart()', () => {` ?